src/tdlib/message.rs

```diff
@@ -139,6 +140,17 @@ mod imp {
                             | glib::ParamFlags::CONSTRUCT
                             | glib::ParamFlags::EXPLICIT_NOTIFY,
                     ),
+                    glib::ParamSpecInt::new(
+                        "edit-date",
+                        "Edit Date",
+                        "Point in time (Unix timestamp) when the message was edited",
+                        std::i32::MIN,
+                        std::i32::MAX,
+                        0,
+                        glib::ParamFlags::READWRITE
+                            | glib::ParamFlags::CONSTRUCT
+                            | glib::ParamFlags::EXPLICIT_NOTIFY,
+                    ),
```

Comment:
I know that this is from tdlib, but I have no idea what's the need of an edit date. Do you happen to know where this may be used? Otherwise I suggest turning this into a boolean for now.

Reply:
No, can't find anything either.